integration-libs/cpq-quote/cpq-quote-discount/components/cpq-quote-discount-tbody/cpq-quote.component.ts

```diff
@@ -52,11 +52,18 @@ export class CpqQuoteDiscountComponent implements OnInit, OnDestroy {
   }
   getDiscountedPrice(
     basePrice: number | undefined,
-    discountPercentage: number | undefined
+    appliedDiscount: number | undefined,
+    quantity: number | undefined
   ): number | undefined {
-    if (basePrice !== undefined && discountPercentage !== undefined) {
-      const discountAmount = (basePrice * discountPercentage) / 100;
-      return basePrice - discountAmount;
+    if (
+      basePrice !== undefined &&
+      appliedDiscount !== undefined &&
+      quantity !== undefined
```

Comment:
'm not completely sure about basePrice and quantity; they should usually be defined here, but I added checks for them just in case. However, appliedDiscount can be undefined because I'm getting data from cpqQuotation.